.github/workflows/py-tests.yml

```diff
@@ -35,6 +35,12 @@ jobs:
         java-version: '11'
         distribution: 'adopt'
 
+    - name: Install open-metadata dependencies
```

Comment:
Can you revert these changes please ? It seems we are trying to run venv before installing. This will fail.

Reply:
correct, I missed to move the installation. However, we need to have the `venv` prepared and the dependencies generated before creating the ingestion image or we won't find the models in there.

With the docker this was not needed because the base container already had the necessary ingredients.